controllers/phish.go

```diff
@@ -195,6 +196,7 @@ func (ps *PhishingServer) ReportHandler(w http.ResponseWriter, r *http.Request)
 // (such as clicked link, etc.)
 func (ps *PhishingServer) PhishHandler(w http.ResponseWriter, r *http.Request) {
 	r, err := setupContext(r)
+	w.Header().Set("X-Server", "GoPhish") // Useful for checking if this is a GoPhish server (e.g. for campaign reporting plugins)
```

Comment:
Let's move this below the error check. The reason is that I only want to reveal that this is a server running Gophish if a valid `rid` parameter is provided.

Additionally, I have a constant called `ServerName` in `config.go` that's set to `gophish`. We can probably reuse this to keep things consistent, since this is what's set in the `X-Mailer` header when sending emails.

```suggestion
	w.Header().Set("X-Server", config.ServerName) // Useful for checking if this is a GoPhish server (e.g. for campaign reporting plugins)
```